packages/charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -32,20 +32,38 @@ export class Arc extends React.Component<IArcProps, IArcState> {
     const id = this.props.uniqText! + this.props.data!.data.legend!.replace(/\s+/, '') + this.props.data!.data.data;
     const opacity: number = this.props.activeArc === this.props.data!.data.legend || this.props.activeArc === '' ? 1 : 0.1;
     return (
-      <g>
+      <g
+        ref={(e: SVGPathElement) => {
+          this._refCallback(e);
```

Comment:
why not use a real `React.createRef()` instance rather than a lambda here?

Reply:
fixed